buda/entities/status.proto

```diff
@@ -0,0 +1,12 @@
+package buda.entities;
+
+message Status {
```

Comment:
would love to chat with you on this one, just to make sure I have full context 😄